src/components/Search/SearchRouter/SearchRouter.tsx

```diff
@@ -310,6 +311,10 @@ function SearchRouter({onRouterClose, shouldHideInputCaret}: SearchRouterProps)
 
     const modalWidth = shouldUseNarrowLayout ? styles.w100 : {width: variables.searchRouterPopoverWidth};
 
+    const isDataLoaded = useMemo(() => {
+        return (!contextualReportID || contextualReportID !== undefined) && !isLoadingOnyxValue(recentSearchesMetadata) && recentReports.length > 0;
```

Comment:
NAB: I think we don't need `contextualReportID` condition here bcoz currently also that condition truthy for any value.
```suggestion
        return !isLoadingOnyxValue(recentSearchesMetadata) && recentReports.length > 0;
```

Reply:
Why is this a NAB? We should never merge code like this @Pujan92 😒

Reply:
It shouldn't be NAB, my bad 😢